edx_proctoring/tests/test_api.py

```diff
@@ -1073,14 +1199,54 @@ def test_get_studentview_unstarted_timed_exam(self):  # pylint: disable=invalid-
         )
         self.assertNotIn('data-exam-id="%d"' % self.proctored_exam_id, rendered_response)
         self.assertIn(self.timed_exam_msg % self.exam_name, rendered_response)
+        self.assertIn('1 hour and 30 minutes', rendered_response)
         self.assertNotIn(self.start_an_exam_msg % self.exam_name, rendered_response)
 
-    def test_get_studentview_completed_timed_exam(self):  # pylint: disable=invalid-name
+    def test_get_studentview_unstarted_timed_exam_with_allowance(self):  # pylint: disable=invalid-name
+        """
+        Test for get_student_view Timed exam which is not proctored and has not started yet.
+        But user has an allowance
+        """
+
+        ProctoredExamStudentAllowance.objects.create(
+            proctored_exam_id=self.timed_exam,
+            user_id=self.user_id,
+            key='Additional time (minutes)',
+            value=15
+        )
+
+        rendered_response = get_student_view(
+            user_id=self.user_id,
+            course_id=self.course_id,
+            content_id=self.content_id_timed,
+            context={
+                'is_proctored': False,
+                'display_name': self.exam_name,
+                'default_time_limit_mins': 90
+            }
+        )
+        self.assertNotIn('data-exam-id="%d"' % self.proctored_exam_id, rendered_response)
```

Comment:
use `.format` instead of `%s` in concatinating strings
`'data-exam-id={proctored_exam_id}'.format(proctored_exam_id=self.proctored_exam_id)`